src/main/resources/application.yml

```diff
@@ -0,0 +1,66 @@
+server:
+  port: 8080
+  shutdown: graceful
+  servlet:
+    context-path: /api
+
+
+# Actuator endpoint configurations
+info:
+  app:
+    version: 0.0.1
+    release: '0_20210610'
+    name: ${spring.application.name}
+    servlet:
+      context-path: /NotificationRelayApp
```

Comment:
Is this your application name?

Reply:
I was facing issues from converting the application.properties to yml format so I used the previous yml file, I have removed this in my next commit.  Now my files are uptodate.